src/Apps/Conversation/Components/Conversations.tsx

```diff
@@ -13,7 +14,7 @@ const Conversations: React.FC<ConversationsProps> = props => {
   const { me } = props
   const conversations = me.conversationsConnection.edges
   return (
-    <Box px={1}>
+    <Box px={1} height="calc(100vh - 180px)" style={{ overflow: "hidden" }}>
```

Comment:
```suggestion
    <Box px={1} height="calc(100vh - 180px)">
```

Who left that there?